### ISSUE TITLE
Can we set "collapse" between specific rows or columns?

### ISSUE THREAD

**Issue by dasiav7** [ropensci/iheatmapr#41]
As I mentioned in the title, can we make a space that separate the specific rows or columns?

**Comment by AliciaSchep** [comment]
No, that is not currently possible with iheatmapr... and I think unfortunately that it would be non-trivial to implement.  Two ways that similar effect might be achieved is 

(1) Break the heatmap into separate heatmaps added together
(2) Add a row or column of NA values 

However both of those approaches could be problematic if wanting to do things like clustering so might not be viable in practice...
